test_regress/t/t_vpi_public_depth.cpp

```diff
@@ -0,0 +1,236 @@
+// -*- mode: C++; c-file-style: "cc-mode" -*-
+//*************************************************************************
+//
+// Copyright 2010-2011 by Wilson Snyder. This program is free software; you can
+// redistribute it and/or modify it under the terms of either the GNU
+// Lesser General Public License Version 3 or the Perl Artistic License
+// Version 2.0.
+// SPDX-License-Identifier: LGPL-3.0-only OR Artistic-2.0
+//
+//*************************************************************************
+
+#ifdef IS_VPI
+
+#include "vpi_user.h"
+
+#include <cstdlib>
+
+#else
+
+#include "verilated.h"
+#include "verilated_vcd_c.h"
+#include "verilated_vpi.h"
+
+#include "Vt_vpi_public_depth.h"
+#include "Vt_vpi_public_depth__Dpi.h"
```

Comment:
This is why you are getting the other error, you need to 
```suggestion
#include VM_PREFIX_INCLUDE
#ifdef T_VPI_PUBLIC_DEPTH
#include "Vt_vpi_public_depth__Dpi.h"
#elif defined(T_VPI_PUBLIC_DEPTH_BAD)
#include "Vt_vpi_public_depth_bad__Dpi.h"
#else
#error "Bad test"
#endif
```